src/chapters/configuring.xml

```diff
@@ -2059,6 +2059,18 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
                                         required if file transfer is enabled.</para>
                                 </entry>
                             </row>
+                            <row>
+                                <entry><parameter>drive-name</parameter></entry>
+                                <entry>
+                                    <para>The name of the filesystem used when passed through to
+                                        the RDP session.  This is the name that users will see in their
+                                        Computer/My Computer area along with client name (for example,
+                                        "Guacamole on Guacamole RDP"), and is also the name of the share
+                                        when accessing the special \\tsclient network location.</para>
```

Comment:
`\\tsclient` should be wrapped in a [`<filename>` tag](https://tdg.docbook.org/tdg/5.0/filename.html).

Reply:
Tagged.